test/integration/retryable-reads/retryable_reads.prose.spec.test.ts

```diff
@@ -0,0 +1,80 @@
+import { expect } from 'chai';
+import * as semver from 'semver';
+
+import { TopologyType } from '../../../src';
+
+const VALID_TOPOLOGIES = [
+  TopologyType.ReplicaSetWithPrimary,
+  TopologyType.Sharded,
+  TopologyType.LoadBalanced
+];
+
+describe('Retryable Reads (prose)', function () {
```

Comment:
See my comment on the retryable_reads.spec.test.js file - I think we should reserve this file for the lone prose test that will be implemented as part of avoiding connection storms